watertap/unit_models/zero_order/tests/test_mabr_zo.py

```diff
@@ -187,21 +182,22 @@ def test_report(self, model):
 
     Variables: 
 
-    Key                                   : Value     : Fixed : Bounds
-                        Blower Size (m^2) :    1.0000 :  True : (0, None)
-                       Electricity Demand :   0.57380 : False : (0, None)
-                    Electricity Intensity : 0.0069300 :  True : (None, None)
-    Reaction Extent [ammonium_to_nitrate] :    1.4000 : False : (None, None)
-    Solute Removal [ammonium_as_nitrogen] :    0.0000 :  True : (0, None)
-                     Solute Removal [bod] :    0.0000 :  True : (0, None)
-                 Solute Removal [nitrate] :    0.0000 :  True : (0, None)
-                     Solute Removal [tss] :    0.0000 :  True : (0, None)
-                           Water Recovery :    1.0000 :  True : (1e-08, 1.0000001)
+    Key                                   : Value      : Fixed : Bounds
+                              Blower Size : 3.9877e+07 : False : (0, None)
```

Comment:
I'm not sure about this blower size in m**2. Mathematically, it works out in the relationships that we have, but what area is this really? I guess it depends on the `air_flow_rate` parameter. Does it represent volume of air flowrate required per effective area of the tank or reactor receiving the air? The way I see it, it can only be that or the effective cross-sectional area of the blower outlet, which seems unlikely due to the large size.

Reply:
Since we know at the least that this just serves as a sizing parameter for getting the blower flowrate, maybe we should leave it out of the report unless we rename `blower size` to something else. Perhaps reactor area?

Reply:
The blower size is the sizing variable for calculating blower cost. You are correct that it represents the volume of air flowrate required per effective area of the reacotr receiving the air. I calculated this value by the nitrogen removal flowrate as suggested, but I don't know if this large value make any sense. Probably I rename it as `reactor area` and doesn't report it.